### PR TITLE
feat(learning): Integrate GLTorch into GraphScope

### PR DESCRIPTION
**Pull request by LiSu**
This PR integrates [GLTorch](https://github.com/alibaba/graphlearn-for-pytorch) into GraphScope, allowing training GNNs using GLTorch in the local mode of GraphScope.

### CODE REVIEW

File: python/setup.py [hunk 1]
```diff
@@ -111,12 +127,54 @@ def run(self):
...
+    def _get_gcc_use_cxx_abi(self):
+        if hasattr(self, "_gcc_use_cxx_abi"):
+            return self._gcc_use_cxx_abi
+        output = subprocess.run(
+            [shutil.which("cmake"), "."],
+            cwd=glt_root_path,
+            capture_output=True,
+            text=True,
+        )
+        import re
```
Comment (sighingnow) [inline python/setup.py:158]: Maybe running cmake in a clean build directory would be better, i.e.,

```
build_dir = os.path.join(glt_root_path, 'cmake-build')
os.makedirs(build_dir, exist_ok=True)
output = subprocess.run([shutil.which('cmake'), '..'], cwd=build_dir, ....)
```

**Review by sighingnow — DISMISSED** [review]
LGTM.

File: python/setup.py (hunk 1 above)
Comment (LiSu) [reply python/setup.py:160]: Fixed by adding a clean cmake build directory.

**Review by sighingnow — APPROVED** [review]
LGTM.

The failure in Python test is irrelevant with this PR.

**Pull request merged**